tests/unit/test_cli.py

```
@@ -223,6 +225,65 @@ def test_cli_context_window_override_environment_variable() -> None:
                 del os.environ["FORCE_CONTEXT_WINDOW"]
 
 
+def test_apply_cli_args_derives_anthropic_port_when_missing() -> None:
+    """Anthropic compatibility port should default to main port + 1."""
+
+    def fake_load_config(
+        _config_path: Any | None = None,
+        *,
+        resolution: ParameterResolution | None = None,
+        environ: Any | None = None,
```

Ruff (0.13.3)

Unused function argument: `environ`

(ARG001)